internal/ad/ad_test.go

```diff
@@ -1311,3 +1315,21 @@ func assertEqualPolicies(t *testing.T, expected policies.Policies, got policies.
 
 	testutils.CompareTreesWithFiltering(t, gotAssetsDir, expectedAssetsDir, false)
 }
+
+func standardUserGPO(id string) policies.GPO {
```

Comment:
This makes a lot of sense sense, thanks for changing and adapting existing test cases too :)